src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -198,6 +198,66 @@ private[spark] object Config {
       .stringConf
       .createOptional
 
+  val DEFAULT_SPARK_EXECUTOR_CORES = "1"
+  val DEFAULT_CORES                = "1"
+  val ARMADA_DRIVER_LIMIT_CORES: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.driver.limit.cores")
+      .doc("Specify the hard cpu limit for the driver pod")
+      .version("1.0.0")
+      .stringConf
```

Comment:
If we do not have to support the milli-notation (`100m`), we can use double here (`0.1`):
```suggestion
      .doubleConf
```
If we really have to support this, it should be mentioned in the description.

Reply:
Again, this is exactly what the k8s implementation does, so I'd prefer to leave it: 

https://github.com/apache/spark/blob/master/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala#L287-L293

Reply:
I will update the description